static/js/modules/nootils.js

```diff
@@ -19,8 +19,8 @@ module.exports = function(settings) {
       } else {
         result = new Date();
       }
-      result.setUTCDate(result.getUTCDate() + days);
-      result.setUTCHours(0, 0, 0, 0);
```

Comment:
This is the only change that matters. Setting the non-utc hours should mean, since we're in the right timezone by default, that this works right.